how_to_test_s3_backend

```diff
@@ -0,0 +1 @@
+sh run_tests.sh --engine cpu --backend s3 --connection value_ignored --connection-options endpoint_url=https://ceph-res-gw1.fds.rl.ac.uk verify=/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
```

Comment:
Once the tests are created (and running in travis), I don't think this file is needed
(it could be worth expanding README.md to specify use cases/examples)